ex6/index.js

```diff
@@ -0,0 +1,37 @@
+const prompt = require("prompt-sync")({ sigint: true});
+
+console.log("Insert side A: ");
+const sideA = prompt(">");
+
+console.log("Insert side B: ");
+const sideB = prompt(">");
+
+console.log("Insert side C: ");
+const sideC = prompt(">");
+
+ function verify() {
```

Comment:
Verify is a very generic name and it is hard to tell what it is verifying. Can you please come up with a better name?

Reply:
I think `function checkIfItIsATriangle()` its a better one.